### PR TITLE
trivial: simple_client: add a refresh metadata function

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: contrib/firmware_packager/simple_client.py [hunk 1]
```diff
@@ -62,13 +62,23 @@ def parse_args():
...
     parser.add_argument('deviceid', nargs='?', help='DeviceID to operate on(optional)')
     args = parser.parse_args()
     return args
 
+def refresh(client):
+    """Uses fwupd client to refresh metadata"""
+    remotes = client.get_remotes()
+    client.set_user_agent_for_package("simple_client", client.get_daemon_version())
+    for remote in remotes:
+        if (not remote.get_enabled()):
```
Comment (hughsie) [inline contrib/firmware_packager/simple_client.py:77]: No outer brackets

File: contrib/firmware_packager/simple_client.py [hunk 2]
```diff
@@ -62,13 +62,23 @@ def parse_args():
...
+            continue
+        if (remote.get_kind() != Fwupd.RemoteKind.DOWNLOAD):
```
Comment (hughsie) [inline contrib/firmware_packager/simple_client.py:79]: Same

File: contrib/firmware_packager/simple_client.py [hunk 3]
```diff
@@ -62,13 +62,23 @@ def parse_args():
...
     )
     parser.add_argument('cab', nargs='?', help='CAB file')
     parser.add_argument('deviceid', nargs='?', help='DeviceID to operate on(optional)')
     args = parser.parse_args()
     return args
 
+def refresh(client):
+    """Uses fwupd client to refresh metadata"""
+    remotes = client.get_remotes()
+    client.set_user_agent_for_package("simple_client", client.get_daemon_version())
```
Comment (hughsie) [inline contrib/firmware_packager/simple_client.py:75]: This is meant to be the front-end version, not the Daemon version.

Comment (superm1) [reply contrib/firmware_packager/simple_client.py:75]: We don't exactly have a version associated with `simple_client.py`.  Thoughts what to put there?

Comment (hughsie) [reply contrib/firmware_packager/simple_client.py:75]: It needs to be the version from the top-level meson.build, I guess we just add a VERSION to the unbuilt file.

**Comment by hughsie** [comment]
@superm1 do you want this in 1.5.3? or just NTH?

**Comment by superm1** [comment]
> @superm1 do you want this in 1.5.3? or just NTH?

No rush on it.  If you can fix it up sure.  I'm low on cycles to do it though myself soon.

**Comment by hughsie** [comment]
@superm1 I fixed this up for you, looks okay now?

**Review by superm1 — COMMENTED** [review]
Yeah looks good thanks

**Pull request merged**